manifests/release/kubevirt.yaml.in

```diff
@@ -5,6 +5,8 @@ metadata:
   labels:
     kubevirt.io: ""
   name: {{.Namespace}}
+spec:
+  permitBridgeInterfaceOnPodNetwork : "false"
```

Comment:
This can be native boolean `false`. We should convert it to string internally, before passing to the config map.

In any case. It should be `true` by default (U/S) and not a required parameter in the spec. If someone wants to disable it, they need to explicitly set the spec. Otherwise it is enabled.